### PR TITLE
Italian data file added

### PR DESCRIPTION
**Pull request by axamon**
It could be more convenient to add language data on diffent files in order to import only needed ones.

### CODE REVIEW

**Comment by aaaton** [comment]
That is not a bad idea. It could reduce the memory footprint of the loaded golem drastically. I will look into how to do it properly.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +0,0 @@
-data
```
Comment (aaaton) [inline .gitignore:1]: Why is this file deleted?

File: main_test.go [hunk 2]
```diff
@@ -21,6 +22,17 @@ func TestReadBinary(t *testing.T) {
 	}
 }
 
+func TestReadBinary_IT(t *testing.T) {
```
Comment (aaaton) [inline main_test.go]: Can you add tests for a couple of italian words as well for some sanity checking?

**Review by aaaton — CHANGES_REQUESTED** [review]

File: main_test.go (hunk 2 above)
Comment (axamon) [reply main_test.go]: I made a test that is multilanguage, we just need to put some more examples in other languages.

**Comment by aaaton** [comment]
First of all: thank you for your hard work Alberto.

I'm closing this pull-request because it has spread to include a bit of everything, and I don't agree with all the changes.

You have had lots of valuable suggestions, so the changes I agree with have been added to v2.0. I have also included a Contributors Hall of Fame in the Readme, where you are included.
